pkg/network/opencontrail/network.go

```diff
@@ -123,6 +118,65 @@ func (m *NetworkManagerImpl) DeleteFloatingIpPool(network *types.VirtualNetwork,
 	return nil
 }
 
+// Subnets can only be deleted if there are no instance-ips or floating-ips associated
+// with the subnet.
+func (m *NetworkManagerImpl) updateSubnetConfig(network *types.VirtualNetwork, prefix string) {
```

Comment:
kube-network-manager only support a single prefix for the public network config. That is ok for now.